### PR TITLE
Tests: Improve SQL wrapper tests

### PR DESCRIPTION
**Pull request by mia-pi-git**

### CODE REVIEW

**Comment by AnnikaCodes** [comment]
Can you address the merge conflicts here?

**Comment by Zarel** [comment]
@mia-pi-git this is still waiting on you.

**Comment by mia-pi-git** [comment]
Ok, handled.

File: test/lib/sql.js [hunk 1]
```diff
@@ -24,12 +25,10 @@ describe(`SQLite worker wrapper`, () => {
...
 	it(`should support both inline and object params`, async () => {
-		const num = await database.prepare(`INSERT INTO test (col, col2) VALUES($col, $col2)`);
 		await database.run(insert, ['a', 'b']);
-		await database.run(num, {col: 'a', col2: 'b'});
+		await database.run(`INSERT INTO test (col, col2) VALUES($col, $col2)`, {col: 'a', col2: 'b'});
 	});
 	it(`should retrieve one line from Database.get`, async () => {
-		const result = await database.get(select);
-		assert(!!result, 1);
+		assert(!!await database.get(select), 1);
```
Comment (Zarel) [inline test/lib/sql.js:32]: What does `1` mean here? The second arg to `assert` is a description of what the bool means, isn't it?

**Comment by Zarel** [comment]
(Also, have you. noticed the build error?)

**Comment by monsanto** [comment]
Old PR, I don't know if it is still relevant. If it is, it needs to be rebased.

**Pull request closed without merging**